pkg/agent/controller/egress/egress_controller.go

```diff
@@ -223,8 +266,22 @@ func (c *EgressController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	go c.cluster.Run(stopCh)
```

Comment:
`c.cluster`should be constructed and started in same place. Otherwise it looks like it's constructed in cmd package but never started.

Reply:
done